lib/tbot/cli/keypair_create.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cli
+
+import "github.com/gravitational/teleport"
+
+// KeypairCreateCommand handles `tbot keypair create`
+type KeypairCreateCommand struct {
+	*genericExecutorHandler[KeypairCreateCommand]
+
+	ProxyServer string
+	Storage     string
+	Overwrite   bool
+	Format      string
+}
+
+// NewKeypairCreateCommand initializes the `keypair create` command and returns
```

Comment:
I'd be tempted to use a more semantic command than "create" here - but nothing immediately comes to mind. I'll mull it over.

Reply:
Possible alternatives:
- `tbot keypair generate`
- `tbot keypair new`
- other suggestions?

I have no strong feelings on the UX here, so let me know what you think!

Reply:
Yeah - perhaps "generate" is nicer. My only other worry is that "keypair" alone may not indicate sufficiently that this is something to do with bound keypair joining - and `tbot` does a lot with keypairs!

Reply:
Per discussion in our 1x1 - I've hidden this for now but left it otherwise unchanged. We may still opt to change the name here before the feature is released.